### PR TITLE
HEAP-38339 added no emotion css instance on component classname rule

### PR DESCRIPTION
**Pull request by xiyang786926**

### CODE REVIEW

**Comment by xiyang786926** [comment]
Screen Shot 2023-01-06 at 11 49 31 AM

File: src/rules/no-emotion-css-instance-classname.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { createRule } from '../utils/createRule';
+
+type MessageIds = 'noEmotionCSSInstanceClassname';
+
+const MESSAGE = 'emotion css instance should not use for classname, use emotion variable instead';
```
Comment (djsaun) [inline src/rules/no-emotion-css-instance-classname.ts]: Can we please reword this to something like "Emotion's css property should not be used directly within a React component. Please move the css instance to a variable outside of the component's scope before using."

**Review by djsaun — CHANGES_REQUESTED** [review]

**Review by greghuels — APPROVED** [review]
